group_project_v2/mixins.py

```diff
@@ -160,22 +327,18 @@ def workgroup(self):
         workgroup = self._get_workgroup(self.project_api, self.user_id, self.course_id)
         return workgroup if workgroup else self.FALLBACK_WORKGROUP
 
-    @staticmethod
-    @memoize_with_expiration()
-    def _get_workgroup(project_api, user_id, course_id):
+    def _get_workgroup(self, project_api, user_id, course_id):
         """
         :rtype: WorkgroupDetails
```

Comment:
@jbzdak yes, this was the idea - memoize the underlying api call and merge it back into property.